R/compare_models.R

```diff
@@ -48,13 +49,9 @@
 
    safe_eval <- purrr::safely(evaluate_model)
 
-   if(is.null(serial_interval)) {
```

Comment:
We might want to keep that as long as we still force to do rts and incidences in one go. Otherwise you will have an error later on after quite a long run time.

Reply:
depending on whether compare_models is also meant as a standalone function.

Reply:
Suggest testing with a small number of samples is probably the best bet here.